mxcube3/core/models/configmodels.py

```diff
@@ -16,6 +16,13 @@ class FlaskConfigModel(BaseModel):
     USER_DB_PATH: str = Field("/tmp/mxcube-user.db", description="")
     PERMANENT_SESSION_LIFETIME: datetime.timedelta
     SESSION_PERMANENT: bool = Field(True, description="")
+    CERT_KEY: str = Field("", description="Full path to signed certficate key file")
```

Comment:
should not these be Optional?

Reply:
They are not used as long CERT is not set to SIGNED, the default value is the empty string so that should be alright. I somehow assumed that one knows that they need to have a value of one set CERT SIGNED.